chains/ethereum/config/src/lib.rs

```diff
@@ -20,7 +20,7 @@ pub fn config(network: &str) -> Result<BlockchainConfig> {
             currency_symbol: "ETH",
             currency_decimals: 18,
             node_uri: NodeUri::parse("http://127.0.0.1:8545")?,
-            node_image: "ethereum/client-go:v1.10.26",
+            node_image: "ethereum/client-go:v1.12.2",
```

Comment:
Can this be configurable at startup?

Reply:
I'm working on it, in another PR I'm refactoring all the tests for using docker-compose.